cognite/client/_api/workflows.py

```diff
@@ -98,6 +99,47 @@ def create(
         )
         return WorkflowTrigger._load(response.json().get("items")[0])
 
+    # TODO: remove method and associated data classes in next release
+    def create(
+        self,
+        workflow_trigger: WorkflowTriggerCreate,
+        client_credentials: ClientCredentials | dict | None = None,
+    ) -> WorkflowTrigger:
+        """`[DEPRECATED] Create or update a trigger for a workflow. <https://api-docs.cognite.com/20230101/tag/Workflow-triggers/operation/CreateOrUpdateTriggers>`_
+
+        This method is deprecated, use '.upsert' instead. It will be completely removed October 2024.
```

Comment:
Check out `DatapointsAPI.retrieve_dataframe_in_tz` for a deprecation warning that stands out